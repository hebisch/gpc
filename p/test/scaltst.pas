program test(output);

var
i :integer;
j :boolean;
t :real;

begin
i := 1;
t := i;
writeln({$if False} Real(i) -- not allowed anymore Frank, 20030317 {$else} t {$endif}:6:2);
j := TRUE;
writeln(false);
writeln(j);
end.
