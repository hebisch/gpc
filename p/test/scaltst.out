  1.00
False
True
